release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -66,9 +66,9 @@ On new clusters, LUKS configuration must use the native Ignition mechanism, as p
 With `bootupd`, {op-system} users now have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures.
 
 [id="ocp-4-7-rhcos-rhel-8-3-packages"]
-==== {op-system} now supports RHEL 8.3
+==== {op-system} now uses RHEL 8.4
 
-{op-system} is now using Red Hat Enterprise Linux (RHEL) 8.3 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
+{op-system} is now using Red Hat Enterprise Linux (RHEL) 8.4 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
```

Comment:
```suggestion
{op-system}  4.7.24 and above now use Red Hat Enterprise Linux (RHEL) 8.4 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
```

Reply:
Thanks, @sdodson! I incorporated your suggestion and am awaiting 4.7.24 GA to merge this content.